src/main/kotlin/uk/gov/justice/digital/hmpps/manageusersapi/resource/prison/SyncController.kt

```diff
@@ -0,0 +1,61 @@
+package uk.gov.justice.digital.hmpps.manageusersapi.resource.prison
+
+import io.swagger.v3.oas.annotations.Operation
+import io.swagger.v3.oas.annotations.Parameter
+import io.swagger.v3.oas.annotations.media.Content
+import io.swagger.v3.oas.annotations.media.Schema
+import io.swagger.v3.oas.annotations.responses.ApiResponse
+import io.swagger.v3.oas.annotations.responses.ApiResponses
+import org.springframework.security.access.prepost.PreAuthorize
+import org.springframework.validation.annotation.Validated
+import org.springframework.web.bind.annotation.PathVariable
+import org.springframework.web.bind.annotation.PostMapping
+import org.springframework.web.bind.annotation.RestController
+import uk.gov.justice.digital.hmpps.manageusersapi.config.ErrorResponse
+import uk.gov.justice.digital.hmpps.manageusersapi.service.prison.SyncService
+
+@RestController
+@Validated
+class SyncController(
+  private val syncService: SyncService,
+) {
+  @PostMapping("/prisonusers/{username}/email/sync")
```

Comment:
Created new sync controller as ideal here (explicit) and as we move forward, we may have something like this for external users